smart_contracts/contracts/client/add-bid/src/main.rs

```diff
@@ -12,6 +12,7 @@ fn add_bid(
     delegation_rate: auction::DelegationRate,
     minimum_delegation_amount: Option<u64>,
     maximum_delegation_amount: Option<u64>,
+    reserved_spots: Option<u32>,
```

Comment:
@EdHastingsCasperAssociation do you think this should be an Option with default value = 0? I was told there was some discussion on the topic before and there's some nuance to that.

Reply:
@EdHastingsCasperAssociation could you please respond and give a final go to this PR?